src/components/ha-button-menu.ts

```diff
@@ -34,6 +37,7 @@ export class HaButtonMenu extends LitElement {
   }
 
   protected render(): TemplateResult {
+    this.style.display = this.hass.enableKioskMode ? "none" : "inline-block";
```

Comment:
Why is this being add to the Component for a case for an element that uses this component?